### PR TITLE
New package: ShiningLight.OpenSSL version 1.0.0

### PR DESCRIPTION
**Pull request by quhxl**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25208)

### CODE REVIEW

**Review by OfficialEsco — APPROVED** [review]

File: manifests/s/ShiningLight/OpenSSL/1.0.0/ShiningLight.OpenSSL.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  Scope: machine
+  InstallerUrl: https://slproweb.com/download/Win64OpenSSL-1_1_1k.msi
+  InstallerSha256: C21E1B472C7777C7B2017FD335FC827976944BB1349731989C8D9E16D913519A
+  UpgradeBehavior: install
+  ProductCode: '{DE02BC89-4A1D-4B5D-A673-DE699BE1119F}'
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://slproweb.com/download/Win32OpenSSL_Light-1_1_1k.msi
```
Comment (ryfu-msft) [inline manifests/s/ShiningLight/OpenSSL/1.0.0/ShiningLight.OpenSSL.installer.yaml:26]: Hey @quhxl,

Is this the correct installerURL for this package? It seems to be pointing to ShiningLight.OpenSSLLight and not OpenSSL. Can you verify or make this change? I believe the correct url should be: https://slproweb.com/download/Win32OpenSSL-1_1_1k.msi

Thanks!

**Review by ryfu-msft — CHANGES_REQUESTED** [review]

**Comment by quhxl** [comment]
If somebody requests the changes with a suggestion this would pass. I am currently not able to change this because I don't have access to my computer rn.

File: manifests/s/ShiningLight/OpenSSL/1.0.0/ShiningLight.OpenSSL.installer.yaml (hunk 1 above)
Comment (OfficialEsco) [reply manifests/s/ShiningLight/OpenSSL/1.0.0/ShiningLight.OpenSSL.installer.yaml:26]: Both urls got `404 Not Found` 👌

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by quhxl** [comment]
I am back!

**Review by ItzLevvie — APPROVED** [review]
image

**Review by ryfu-msft — APPROVED** [review]

**Pull request merged**